src/Tribe/Plugins_API.php

```diff
@@ -0,0 +1,310 @@
+<?php
+
+class Tribe__Plugins_API {
+	/**
+	 * Static Singleton Factory Method
+	 *
+	 * @since tbd
```

Comment:
the `@since tbd`s are clearly intentional, but any specific plan for changing them? 

 is this part of the new process Brook proposed of doing this all the time?